winch/pac_test.go

```diff
@@ -0,0 +1,79 @@
+package winch
```

Comment:
not sold on this test. Feels brittle and we are just testing our template here. I think you can kill this.